bug_report_page.php

```diff
@@ -230,16 +230,21 @@
 layout_page_begin( __FILE__ );
 
 $t_form_encoding = '';
+$t_form_class = '';
 if( $t_show_attachments ) {
 	$t_form_encoding = 'enctype="multipart/form-data"';
+	$t_form_class = 'class="dropzone-form"';
 }
 ?>
 <div class="col-md-12 col-xs-12">
 <form id="report_bug_form"
 	method="post" <?php echo $t_form_encoding; ?>
 	action="bug_report.php?posted=1"
-	class="dropzone-form"
-	<?php print_dropzone_form_data() ?>>
+	<?php echo $t_form_class ?>
```

Comment:
Here are a couple of suggestions for approaching this:

1. Change `print_dropzone_form_data` to `prepare_dropzone_form_data()` and capture its output into a variable like the class in this case.  This way the code for outputting the form is easier to read without having IF statements in the middle of a `form` tag.

2. Change `print_dropzone_form_data()` to take `$t_show_attachments` as a parameter and based on that decide to output the class and the other fields.  I know that can cause two class attributes if the form has a class already, but in both cases where we use drop zone, that is not the case.